app/controllers/application_controller.rb

```diff
@@ -52,8 +51,15 @@ def get_commit
     end
   end
 
+  def menues
+    menu = Menu.where(visible: true)
+    @menu_sektionen = menu.where(location: "Sektionen")
+    @menu_medlemmar = menu.where(location: "För medlemmar")
+    @menu_foretag = menu.where(location: "För företag")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.